src/backend/fs.rs

```diff
@@ -31,6 +33,7 @@ pub async fn write_file<'a>(
     pk: &Secp256k1PubKey,
     file_stream: FileStream,
     name: Option<String>,
+    mime_type: &str,
```

Comment:
```suggestion
```
Let's not pass it in here, but instead, set it from the new `(Bytes, String)` tuple here:
https://github.com/diba-io/carbonado-node/pull/52/files#diff-d731782a4b2603919d38af2315cedf79f4d9ed9e5d1b4a1c8b23d74a75f337e8R57